coordinator/persistence-api/src/main/java/io/stargate/db/schema/VectorType.java

```diff
@@ -30,6 +30,11 @@ public String marshalTypeName() {
     return marshalTypeName;
   }
 
+  @Override
```

Comment:
We need access to element type, with Cassandra 5.0 vector type is adding other data type support.

Reply:
Yes, eventually. But is it needed now? As in, do we know how it'd be used? My only concern is with API, translating element type needs to be done carefully.